roxie/roxiemem/roxiemem.cpp

```diff
@@ -166,6 +170,28 @@ static void initializeHeap(bool allowHugePages, bool allowTransparentHugePages,
 #else
     heapUseHugePages = false;
 
+#ifdef _USE_NUMA
+    int has_numa = numa_available() + 1;
+    if (has_numa)
```

Comment:
Personally I'd probably code it as 
  bool has_numa = numa_available() != -1;
  if (has_numa)
  ...

just seems clearer ...